docs/software/programming.md

```diff
@@ -67,4 +68,25 @@ Source: [User:`Maklaan` - Based on a Docker blog post](https://commons.wikimedia
 
 See also <https://wikipedia.org/wiki/Docker_(software)> resp. <https://docs.docker.com/get-started/overview>.
 
+## Portainer
+
+Portainer simplifies your Docker container management via Portainer web interface. It enables faster deploy of the applications and it gives real time visibility.
+
+![DietPi Optimised Software Portainer](../assets/images/dietpi-software-portainer.jpg)
+
+=== "Quick start"
+
+    - Start Portainer using next url in the browser (`https://<your.local.ip>:9002` or `https://<your.host.name>:9002`)[^1].
```

Comment:
it's rechargeable via http

Reply:
fixed in https://github.com/MichaIng/DietPi-Docs/pull/284